reference/src/taimpl/src/ta_sa_key_generate.c

```diff
@@ -154,7 +154,7 @@ static sa_status ta_sa_key_generate_rsa(
     *key = INVALID_HANDLE;
 
     if (rights == NULL) {
-        ERROR("NULL parameters");
```

Comment:
Looks like this file needs the copyright year extension.

Reply:
Done